_projects/htbac.md

```diff
@@ -5,9 +5,21 @@ permalink: /projects/htbac
 abstract: "Enabling rapid, accurate, scalable and reliable free energy-based
 binding affinity calculations." 
 status: active
+logo: htbac_logo.jpg
 ---
 
-This is a collaborative project that studies a wide range of cancer drugs and candidate ligands. The goal is to support personalized clinical decision-making based on genome sequencing and drug discovery. We use ensemble-based, high-performance computing methods, running automated workflows at unprecedented scales. We developed the [High-Throughput Binding Affinity Calculator (HTBAC)](https://github.com/radical-cybertools/htbac), integrating of the [Binding Affinity Calculator (BAC)](https://pubs.acs.org/doi/10.1021/ci8000937), developed by the [Centre for Computational Science](http://ccs.chem.ucl.ac.uk/) at UCL, with [RADICAL-Cybertools](https://radical-cybertools.github.io/), developed by the RADICAL Laboratory at Rutgers University. HTBAC enables rapid, accurate, scalable, reliable and adaptive free energy-based binding affinity calculations, separating the definition of a free energy protocols from its distributed execution. Adaptivity allows users to change the course of execution during runtime, favoring interesting simulations and drug candidates. Co-PIs on the project are Peter Coveney (UCL) and Shantenu Jha (Rutgers).
-
-<a href="https://github.com/radical-cybertools/htbac"><i class="fa fa-github"></i>GitHub Repository</a><br>
-<a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1713749"><i class="fa fa-nsf"></i>NSF Award</a>
+<div class="project-description">
+	<figure>
+		<img src="../images/projects/htbac.png" width="400">
```

Comment:
Here the user will not need to add the exact name. The page can pick it up from the header
```suggestion
		<img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
```